__tests__/app.test.js

```diff
@@ -105,6 +105,25 @@ describe("/api/articles", ()=>{
       expect(body.articles).toBeSortedBy("created_at", { descending: true });
     });
   });
+  test("200: responds with array of articles filtered by the topic mitch", ()=>{
+    return request(app)
+    .get("/api/articles?topic=mitch")
+    .expect(200)
+    .then(({ body })=>{
+      expect(body.articles[0].topic).toBe("mitch");
+      expect(Array.isArray(body.articles)).toBe(true);
```

Comment:
You could go a step further with this test, would we expect only the first article in the response to have the topic `mitch` or would we expect them all to have the topic `mitch`?
Would we expect them to also still contain all their other properties ?